ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/TopicPartitionEntity.java

```diff
@@ -72,4 +76,19 @@ public String toString() {
         + ", partition=" + partition
         + '}';
   }
+
+  @JsonIgnoreProperties(ignoreUnknown = true)
+  public static class TopicPartitionEntityDeserializer extends KeyDeserializer {
+    @Override
+    public TopicPartitionEntity deserializeKey(
+        final String key,
+        final DeserializationContext ctxt
+    ) throws IOException {
+      final String[] split = key.split("=");
```

Comment:
If anyone has a fancy regex solution I would prefer that, this was just a quick workaround